### PR TITLE
fix: fix start script

### LINKED ISSUE #1108: start failed if pid file is exist
WX20220117-183718@2x

### PR DESCRIPTION
**Pull request by dl239**
fix #1108

### CODE REVIEW

File: release/bin/start.sh [hunk 1]
```diff
@@ -83,7 +83,10 @@ case $OP in
         then
              echo "no $COMPONENT to stop (could not find file $OPENMLDB_PID_FILE)"
         else
-            tr -d '\0' < "$OPENMLDB_PID_FILE" | xargs kill
+            PID="$(tr -d '\0' < "$OPENMLDB_PID_FILE")"
+            if kill -0 "$PID" > /dev/null 2>&1; then
+                kill "$PID"
```
Comment (tobegit3hub) [inline release/bin/start.sh:88]: Try `kill -9` to make sure it is killed?

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: release/bin/start.sh (hunk 1 above)
Comment (dl239) [reply release/bin/start.sh:88]: `mon` will not kill its subprocess if use `kill -9`

**Review by aceforeverd — APPROVED** [review]
It just work.

But not robust, if reboot happens the old pid file exists in fs, and another process lives just with that pid number.

**Comment by dl239** [comment]
> It just work.
> 
> But not robust, if reboot happens the old pid file exists in fs, and another process lives just with that pid number.

yes, do you have some advice about this?  There may be another OpenMLDB process with the same pid

**Pull request merged**